datastore/src/test/scala/org/allenai/datastore/DatastoreUtilsSpec.scala

```diff
@@ -0,0 +1,61 @@
+package org.allenai.datastore
+
+import org.allenai.common.testkit.UnitSpec
+
+import com.typesafe.config.ConfigFactory
+
+import scala.collection.JavaConverters._
+
+class DatastoreUtilsSpec extends UnitSpec {
```

Comment:
Since this downloads from S3, it should probably be marked as an integration test.